tpu-provisioner/internal/cloud/gke.go

```diff
@@ -48,8 +49,9 @@ type GKE struct {
 
 	Recorder record.EventRecorder
 
-	inProgressDeletes sync.Map
-	inProgressCreates sync.Map
+	inProgressDeletes       sync.Map
```

Comment:
Can we either update the variable name or add a comment so that it's clear `inProgressDeletes` and `inProgressCreates` store node pool names to do the tracking?